internal/svcmanager/svcmanager.go

```diff
@@ -101,5 +112,10 @@ func (m *Manager) ping(ctx context.Context) error {
 	if err := client.RunWithContext(ctx, r, &resp); err != nil {
 		return err
 	}
+
+	if resp.Version.State.Version != constants.Version && resp.Version.State.Branch != constants.BranchName {
+		return errVersionMismatch{locale.NewError("err_ping_version_mismatch")}
```

Comment:
So far our typed errors have always been pointers IIRC. Prefer we stick with that convention.